components/Select/Select.tsx

```diff
@@ -17,12 +60,26 @@ const StyledSelect = styled('select', {
   pr: '$3',
   lineHeight: '25px',
 
+  '& option': {
+    color: '$loContrast',
```

Comment:
this does not solve dark mode / light mode issue : now in light mode, text has no contrast with the option background:

https://user-images.githubusercontent.com/3367393/140731618-5e80dfbd-5420-45ca-8b4b-75d1e9d0f28f.mp4


The problem is that the select dropdown background does not follow dark/light mode practice : it's always white.

Maybe you should update the select dropdown background when i dark mode, so that it's aligned with the logic.

Here's a working example from mui: https://mui.com/components/selects/#native-select